### DISCUSSION TITLE
Where did you hear about inquire?

### DISCUSSION THREAD

**Discussion by mikaelmello** [mikaelmello/inquire#127] [General]
I really like to see what people are building with this, from the first few users to the several hundreds of stars this repo has today, it makes me really glad that people use and like this library.

2021 was the last time I posted publicly about this (on Reddit), and 700+ stars laters I'm really curious to know how people hear about it. There were even a few sharp increases in stars of which I have no idea where they came from lol

So, where did you hear about this library?

**Comment by danurna** [comment]
Not using it (yet) but it has been the first hit for me googling 'rust cli prompt'. Looking to replace a javascript CLI tool of mine :)

**Comment by jasonish** [comment]
Just discovered this today and it looks like it might work well.  Found it on a Reddit thread after searching for "rust cli menu builder" I think.  I'm building a small hierarchy of menus, currently it just displays a list and you select the number and get the response.  Switching to this will change the flow a little, but I think for the better, and less code to write.
